src/Console/AutomaticInstallation.php

```diff
@@ -93,7 +93,7 @@ protected function getHelp()
 	bin/console autoinstall --savedb
 		Installs Friendica with environment variables and saves them to the 'config/local.config.php' file
 
-	bin/console autoinstall -h localhost -p 3365 -U user -P passwort1234 -d friendica
+	bin/console autoinstall -h localhost -p 3365 -u user -P passwort1234 -d friendica
```

Comment:
It is `-H` not `-h`. I guess you missed this one? :-)

Reply:
nice catch .. fixed